contracts/RandomBeaconHistory.cdc

```diff
@@ -158,6 +325,11 @@ access(all) contract RandomBeaconHistory {
         return self.lowestHeight ?? panic("History has not yet been initialized")
     }
 
+    /// Getter for the contract's Backfiller resource
+    access(all) fun borrowBackfiller(): &Backfiller? {
```

Comment:
We need to update access control here since we added an `access(all)` setter for `maxEntriesPerCall`. I'll push changes here in a minute

Reply:
As mentioned in my previous [comment](https://github.com/onflow/flow-core-contracts/pull/416#discussion_r1552561873), I wanted to confirm that `setMaxEntriesPerCall` can only be called by the service account ?

Reply:
Sorry to have missed that comment. As far as I know, access to `heartbeat()` is mediated by the contract and storage pattern of the `Heartbeat` resource, unless there's some special casing in FVM I'm not aware of. In this case, we'll also want to restrict public access on `Backfiller` due to the `access(all) fun setMaxEntriesPerCall` declaration. The `access(contract)` restriction I made in [a11baea](https://github.com/onflow/flow-core-contracts/pull/416/commits/a11baea87e7fc56232783d5a00100e96a19397e0) should fit the need here.